onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorRegistration.cpp

```diff
@@ -677,29 +678,48 @@ constexpr auto requiredConstantCpuInputs(Args... args)
     return std::make_pair(inputs, static_cast<int>(sizeof...(args)));
 }
 
+template<typename... Args>
+constexpr auto alias(Args... args)
```

Comment:
```suggestion
constexpr auto Aliases(Args... args)
```
Since it appears to supports plural - a list of aliases.